README.md

```diff
@@ -44,21 +44,21 @@ snyk container monitor <repository>:<tag> --org=<ORG_ID>
 **Usage example:**
 
 ```shell
-(base) ➜  python-cache-demo git:(vuln-scanning) snyk code test
+(base) ➜  services git:(vuln-scanning) snyk code test
```

Comment:
I've went through the docs (https://docs.snyk.io/integrate-with-snyk/git-repositories-scms-integrations-with-snyk/snyk-github-integration) and changing the Readme shouldn't break anything. LGMTM!